sys/hashes/sha512_common.c

```diff
@@ -64,6 +63,13 @@ static void be64enc_vect(void *dst_, const void *src_, size_t len)
             dst[i + 7] = src[i];
         }
     }
+    if (i < len) {
```

Comment:
(Again, not an expert on the topic...)
What does it mean when `i < len`?

Reply:
Will add a comment on this.